plugin-scalebar/src/main/java/com/mapbox/maps/plugin/scalebar/ScaleBarPluginImpl.kt

```diff
@@ -114,10 +114,7 @@ open class ScaleBarPluginImpl(
     get() = internalSettings.enabled
     set(value) {
       if (value) {
-        mapListenerDelegate.addOnCameraChangeListener(cameraChangeListener)
         invalidateScaleBar()
-      } else {
-        mapListenerDelegate.removeOnCameraChangeListener(cameraChangeListener)
```

Comment:
why would we remoce thewse add/remove listeners?

Reply:
correct, we do not need it. it was working OK but that's an incorrect solution, I've updated the code